android/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,207 @@
+    <string name="label_unconfirmed">doğrulanmamış</string>
+    <string name="button_receive">Alıcı</string>
+    <string name="button_send">Gönder</string>
+    <string name="button_display_currency">Para Birimi: %1s</string>
+    <string name="label_last_sync">senkronize %1s</string>
+    <string name="label_erg_price">1 ERG ≈</string>
+    <string name="label_fiat_amount">≈ %1s</string>
+    <string name="label_wallet_default">Cüzdanım</string>
+    <string name="label_readonly_wallet_default">Cüzdanım (Sadece Görebilir)</string>
+    <string name="hint_signing_request">Lütfen imzalama talebini ilgili Cüzdanın Para gönder ekranından tarayın.</string>
+    <string name="error_qr_code_content_unknown">Taranan QR kodu bilinen bir formatta değildi.</string>
+
+    <!-- add wallet -->
+    <string name="menu_add_wallet">Cüzdan Ekle</string>
+    <string name="label_create_wallet">Yeni Cüzdan</string>
+    <string name="desc_create_wallet">Rasgele oluşturulmuş bir anımsatıcıdan bir cüzdan (anahtarlar ve adres) oluşturur.</string>
+    <string name="label_restore_wallet">Cüzdanı geri yükle</string>
+    <string name="desc_restore_wallet">Bir cüzdanın anahtarlarını ve adresini anımsatıcı bir ifadeden geri yükler.</string>
+    <string name="label_import_wallet">Cüzdanı İçeri Aktar</string>
+    <string name="desc_import_wallet">Varolan bir cüzdanı (anahtarlar ve adresler) içe aktarmak için dışa aktarılan veya kaydedilmiş bir cüzdan dosyası kullanın.</string>
+    <string name="label_readonly_wallet">Sadece Görülebilir Cüzdan</string>
+    <string name="desc_readonly_wallet">Sadece görülebilir modda bir cüzdan adresi ekleyin. Bu cihazda herhangi bir bilgi saklamayacaktır.</string>
+
+    <!-- add readonly wallet -->
+    <string name="intro_add_readonly">Cüzdanı sadece görülebilir modda eklemek için genel cüzdan adresinizi veya genişletilmiş genel anahtarınızı girin.</string>
+    <string name="label_wallet_address">Cüzdan adresi veya anahtarı</string>
+    <string name="error_address_already_added">Adres zaten eklendi \'%1s\'</string>
+    <string name="error_invalid_readonly_input">Geçerli bir ERG adresi veya ortak anahtar değil</string>
+
+    <!-- Create wallet -->
+    <string name="intro_create_wallet">Bu senin 15 kelimelik Ergo cüzdan anımsatıcın. Bu kelimeler aynı zamanda bir tohum cümlesi veya kağıt anahtarı olarak da bilinir.
+    Bu 15 kelimeyle, buna erişiminizi kaybederseniz cüzdanınızı diğer cihazlara geri yükleyebilirsiniz.\n Tüm 15 kelimeyi (kalem ve kağıtla) dikkatlice yazmanız ve güvenli bir yerde saklamanız çok önemlidir.</string>
+    <string name="intro_confirm_create_wallet">El yazısıyla yazılmış anımsatıcınızın doğru olduğunu doğrulamak için şimdi ifadenin iki kelimesini girin:</string>
+    <string name="check_confirm_create_wallet">Anımsatıcımı güvenli bir yerde saklamaktan sorumlu olduğumu anlıyorum. Eğer bu cihaz üzerinde saklanan anahtarları kayıp edersen, cüzdana ulaşmak için tek yol bu cihazdır.</string>
```

Comment:
<string name="check_confirm_create_wallet">Anımsatıcımı güvenli bir yerde saklamaktan sorumlu olduğumu anlıyorum. Eğer bu cihaz üzerinde saklanan anahtarlar kaybolursa, cüzdanıma erişmenin tek yolu anımsatıcımı kullanmaktır.</string>

Reply:
...anahtarlar kaybolursa, cüzdanıma erişmenin tek yolu anımsatıcımı kullanmaktır.

* Added "anımsatıcımı" word (means mnemonic), the phrase will make more sense that way.